src/candela/components/Geo/index.js

```diff
@@ -0,0 +1,39 @@
+import geojs from 'geojs/geo.js';
+import VisComponent from '../../VisComponent';
+
+export default class Geo extends VisComponent {
+  constructor (el, options) {
+    super(el);
+
+    const geojs_opt = options.geojs || {};
+
+    this.plot = geojs.map(Object.assign({
+      node: el
+    }, geojs_opt));
+
+    this.plot.createLayer('osm', {
+      renderer: null
```

Comment:
Not sure I understand - before creating a point feature we need to create a layer, right? Perhaps @manthey can set me straight.

In any case, the layer should not be created this way in the constructor, which is an oversight on my part. I moved other construction-time options into that `geojs_opt` mechanism and I think we need something similar for creating the base layer. I think for now just setting another top-level option called `baseLayer` or something like that will suffice. I'll make the change and then collect feedback from the GeoJSes :smile_cat: